server/server.js

```diff
@@ -22,16 +33,27 @@ export default function server(app: express, options: any):express {
     app.use("/kue", queueUiRouter({ hostSecret, queueAgent: queue }));
   }
 
-  app.use((req, res, next) => {
+  const applicationDirectory = path.dirname(
+    path.join(require.main.filename, "..")
+  );
+
+  // any subdirectories can serve custom static assets
+  app.use(express.static(`${applicationDirectory}/connectors`));
+
+  const routes = express.Router();
+
+  const validationMiddleware = (req, res, next) => {
     if (req.hull && req.hull.ship) {
       req.agent = new SyncAgent(req.hull);
       return next();
     }
 
     return res.status(403).json({ status: "missing credentials" });
-  });
+  };
 
-  app.get("/admin.html", ({ agent }, res) => {
+  routes.get("/admin.html",
+    validationMiddleware,
+    ({ agent }, res) => {
```

Comment:
had to add the validation middle ware specifically to the routes, otherwise readmes would get caught in it too